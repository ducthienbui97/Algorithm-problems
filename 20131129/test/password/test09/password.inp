123456789 50