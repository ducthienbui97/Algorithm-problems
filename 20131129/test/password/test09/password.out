15866344685926181553959063378512693398678114664448